vispy/visuals/polygon.py

```diff
@@ -38,17 +36,34 @@ class PolygonVisual(Visual):
     """
     def __init__(self, pos=None, color='black',
                  border_color=None, border_width=1, **kwargs):
-        super(PolygonVisual, self).__init__(**kwargs)
-
-        self.mesh = MeshVisual()
```

Comment:
Yes, that looks fine. We generally make attributes into properties when they are considered public, so that's as it should be.